### PR TITLE
Feature hide nav bar on login

### LINKED ISSUE #17: Hide nav bar on login

### PR DESCRIPTION
**Pull request by cmuzzin**
- Showing or hiding the nav bar based on user login status
- Added form validation

### CODE REVIEW

File: src/app/shared/components/nav/nav.component.ts [hunk 1]
```diff
@@ -7,9 +8,13 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NavComponent implements OnInit {
   title = 'Coffee Club';
-  constructor() { }
+  isloggedIn: boolean = false;
```
Comment (toepump) [inline src/app/shared/components/nav/nav.component.ts]: is `isLoggedIn` used?

Comment (cmuzzin) [reply src/app/shared/components/nav/nav.component.ts]: oops, no it isn't

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -11,24 +14,39 @@ export class HomeComponent implements OnInit {
...
+    private router: Router,
+    private fb: FormBuilder,
     public authService: AuthService
-  ) { }
+  ) { 
+    this.form = this.fb.group({
+      email: ['', [Validators.required, Validators.email]],
+      password: ['', [Validators.required]],
+    });
+    this.form.valueChanges.pipe(
```
Comment (toepump) [inline src/app/home/home.component.ts:28]: Is this mainly for debugging?

Comment (cmuzzin) [reply src/app/home/home.component.ts:28]: eventually the forms will be dynamic and that change will fire an update error messages function.

**Review by toepump — APPROVED** [review]

**Pull request merged**